tests/cypress/e2e/lend/basic.cy.ts

```diff
@@ -3,13 +3,12 @@ import { LOAD_TIMEOUT } from '@cy/support/ui'
 describe('Basic Access Test', () => {
   it('should open the Lend DApp successfully', () => {
     cy.visit('/lend/')
-    cy.title(LOAD_TIMEOUT).should('include', 'Lend')
+    cy.url(LOAD_TIMEOUT).should('match', /http:\/\/localhost:\d+\/llamalend\/ethereum\/markets\/?$/)
   })
 
   it('should redirect from the old root URL successfully', () => {
     cy.visit('/lend/#/ethereum')
-    cy.url(LOAD_TIMEOUT).should('match', /http:\/\/localhost:\d+\/lend\/ethereum\/markets\/?$/)
-    cy.title().should('equal', 'Markets - Curve Lend')
```

Comment:
I think it's nice to test the title, why was it removed?

Reply:
Didn't think much of it tbh 🤔